nsxt/resource_nsxt_policy_tier0_gateway.go

```diff
@@ -75,6 +75,7 @@ func resourceNsxtPolicyTier0Gateway() *schema.Resource {
 				Description: "Force whitelisting",
 				Default:     false,
 				Optional:    true,
+				Deprecated:  "Use nsxt_policy_predefined_gateway_policy resource to control default action",
```

Comment:
force_whitelisting should be marked as deprecated in the doc of tier0 and tier1 as well.

Reply:
Good point, done